legacy/dist/preview release/what's new.md

```diff
@@ -417,6 +417,7 @@
 - Fix spherical harmonics computation ([Meakk](https://github.com/Meakk))
 - Fix KTX and DDS loading with baked mipmaps ([Meakk](https://github.com/Meakk))
 - Fix text rendering speed when `TextWrapping.Ellipsis` is used ([carolhmj](https://github.com/carolhmj))
+- Fix issue where type narrowing wasn't working with DeviceType in DeviceSourceManager ([PolygonalSun](https://github.com/PolygonalSun))
```

Comment:
Please remove this from what's new. We didn't ship an incorrect type narrowing in 4.2.